test/ffmpeg-qsv/encode/av1.py

```diff
@@ -37,6 +38,13 @@ def test_r2r(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
     vars(self).setdefault("r2r", 5)
     self.encode()
 
+  @slash.requires(have_string_api)
+  @slash.parametrize(*gen_av1_cqp_parameters(spec))
```

Comment:
Upon second thought, I don't think we should reuse these gen functions because we'll essentially double the number of cases that are already defined for this config variant.  We should introduce a new variant ("strapi"?) in the config files.  This variant could be generalized to include the rcmode parameter similar to the `pict` tests so we don't have to write a test function for every rcmode test class.

Reply:
ok, let me to update and submit another version

Reply:
updated, used dedicated parameter for str api
and firstly added avc/hevc 8bit/av1 encode cqp/cbr/vbr with vme and lp supported firstly